Added latest js project

sgryadav opened the pull request.

sgryadav committed:
Added latest js project

dancinturtle commented:
Hi Sagar, 
Your numbersOnly function is working wonderfully, however you seem to have a bug in your stringsOnly function. Double check your ending condition - you start with i=arr.length-1, but then you are running the function as long as i is less than or equal to 0.

The other problem is that your logic says that you're going to travel backward through the array and pop form the array if you see something that is a number. That means that if my array is [1,2,3,"4", "5"], then i will travel backwards and pop from the array when it sees the number 3. So the string "5" will be popped!

Meanwhile, you submitted jsAssignments.html and numbersOnly.js. However, your html file is trying to import a few other .js files. Without those files, when I run your code, I see multiple errors in the console. That's pretty distracting, so I would remove those lines from your html.

Give this another try and send me another pull request when you have a chance.

sgryadav commented:
Hello Amy,

I have removed the other imports in the html file and fixed the stringsOnly
function.

I have also just sent you another pull request with my updated version.

Thanks for the help!

Best,
Sagar Yadav

On Tue, Oct 24, 2017 at 3:33 PM, Amy <notifications@github.com> wrote:

> Closed #9 <https://github.com/dancinturtle/javaScriptFunOne/pull/9>.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/dancinturtle/javaScriptFunOne/pull/9#event-1308783222>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ANCSb67aZR4IJ2Lp23i8QWYDBGYhlXrnks5svmWngaJpZM4QDnF->
> .
>
